homeassistant/components/overkiz/number.py

```diff
@@ -58,26 +58,26 @@ class OverkizNumberDescription(NumberEntityDescription, OverkizNumberDescription
         name="Eco Room Temperature",
         icon="mdi:thermometer",
         command=OverkizCommand.SET_ECO_TEMPERATURE,
-        min_value=6,
-        max_value=29,
+        native_min_value=6,
+        native_max_value=29,
```

Comment:
Are these temperatures in °C?
If so, we should set device_class to temperature and native_unit_of_measurement to °C.

Reply:
Yes this is Celsius.

Reply:
Yes, there in °C indeed.